diff --git a/launch/rplidar.launch b/launch/rplidar.launch
index aa35534..b362897 100644
--- a/launch/rplidar.launch
+++ b/launch/rplidar.launch
@@ -1,6 +1,21 @@
 <launch>
+<!--
+  <node pkg="hoverbot" name="control" type="control.py" output="screen" >
+  <param name="port" type="string" value="/dev/ttyUSB1"/>
+  </node>
+  <node pkg="turtlesim" name="kb" type="turtle_teleop_key" output="screen" >
+  </node>
+  <node pkg="joy" name="joystick" type="joy_node" output="screen" >
+  </node>
+
+  <node pkg="rosserial_python" type="serial_node.py" name="rosserial" output="screen">
+    <param name="port" value="/dev/ttyUSB0" />
+    <param name="baud" value="115200" />
+  </node>
+-->
+
   <node name="rplidarNode"          pkg="rplidar_ros"  type="rplidarNode" output="screen">
-  <param name="serial_port"         type="string" value="/dev/ttyUSB0"/>
+  <param name="serial_port"         type="string" value="/dev/ttyUSB1"/>
   <param name="serial_baudrate"     type="int"    value="115200"/><!--A1/A2 -->
   <!--param name="serial_baudrate"     type="int"    value="256000"--><!--A3 -->
   <param name="frame_id"            type="string" value="laser"/>
